README.md

```diff
@@ -2,7 +2,12 @@
 
 # Federation API
 
-[![Coverage Status](https://coveralls.io/repos/github/neurobagel/federation-api/badge.svg?branch=main)](https://coveralls.io/github/neurobagel/federation-api?branch=main)
+[![Tests](https://img.shields.io/github/actions/workflow/status/neurobagel/federation-api/test.yaml?color=BDB76B&label=test&style=flat)](https://github.com/neurobagel/federation-api/actions/workflows/test.yaml)
+[![codecov](https://codecov.io/gh/neurobagel/federation-api/graph/badge.svg?token=B827PI9W1U)](https://codecov.io/gh/neurobagel/federation-api)
+[![License](https://img.shields.io/github/license/neurobagel/federation-api?color=CD5C5C&style=flat)](LICENSE)
+[![Python versions](https://img.shields.io/badge/Python-3.10%20%7C%203.11-blue?style=flat)](https://www.python.org)
```

Comment:
```suggestion
[![Python versions](https://img.shields.io/badge/python-3.10-4682B4?style=flat)](https://www.python.org/)
```
we only test 3.10 in the APIs!

Reply:
also, since we're adding badges, how about adding one for the latest-built Docker image version as well? see https://github.com/neurobagel/bagel-cli/blob/5694ebee520699a8cb03b1f12116ccfedce5de4d/README.md?plain=1#L6 for example